src/module/actors/npc.ts

```diff
@@ -29,6 +51,13 @@ export class SWNRNPCActor extends SWNRBaseActor<"npc"> {
     ]);
   }
 }
+// Unsure of right types for TS here.
+Hooks.on("createToken", (document, options, userId) => {
+  if (document.actor?.type == "npc") {
+    document.actor.rollHitDice();
+  }
+});
+
```

Comment:
This needs to be gated behind an option to disable it if people really don't want it. Automation is good, but for some people it pisses them off.

Reply:
wrapped in a new setting (which is on by default)